hack/testing/test-020-olm-upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Given an OLM manifest, verify a green field deployment
+# of cluster logging by asserting CLO creates the resources
+# that begets the operands that make up logging.
+
+set -e
+if [ "${DEBUG:-}" = "true" ]; then
+	set -x
+fi
+
+repo_dir="$( cd "$(dirname "$0")/../.." ; pwd -P )"
+source "$repo_dir/hack/lib/init.sh"
+source "$repo_dir/hack/testing/assertions"
+source "$repo_dir/hack/testing/utils"
+
+start_seconds=$(date +%s)
+os::test::junit::declare_suite_start "${BASH_SOURCE[0]}"
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-"$repo_dir/_output"}
+if [ ! -d $ARTIFACT_DIR ] ; then
+  mkdir -p $ARTIFACT_DIR
+fi
+
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+TIMEOUT_MIN=$((2 * $minute))
+NAMESPACE="openshift-logging"
+manifest_dir=${repo_dir}/manifests
+version=$(basename $(find $manifest_dir -type d | sort -r | head -n 1))
+
+cleanup(){
+  os::log::info "Running cleanup"
+  end_seconds=$(date +%s)
+  runtime="$(($end_seconds - $start_seconds))s"
+  local return_code="$?"
+  set +e
+  oc -n openshift-operators-redhat -o yaml get subscription elasticsearch-operator > $ARTIFACT_DIR/subscription-eo.yml 2>&1 ||:
+  oc logs -n ${NAMESPACE} deployment/cluster-logging-operator > $ARTIFACT_DIR/cluster-logging-operator.log 2>&1 ||:
+  oc -n ${NAMESPACE} -o yaml get subscription cluster-logging-operator > $ARTIFACT_DIR/subscription-clo.yml 2>&1 ||:
+  oc -n ${NAMESPACE} -o yaml get configmap cluster-logging > $ARTIFACT_DIR/configmap-for-catalogsource-clo.yml 2>&1 ||:
+  oc -n ${NAMESPACE} -o yaml get catalogsource cluster-logging > $ARTIFACT_DIR/catalogsource-clo.yml 2>&1 ||:
+  oc  -n openshift-operator-lifecycle-manager logs deployment/catalog-operator > $ARTIFACT_DIR/catalog-operator.logs --since=$runtime 2>&1 ||:
+  oc  -n openshift-operator-lifecycle-manager logs deployment/olm-operator > $ARTIFACT_DIR/olm-operator.logs --since=$runtime 2>&1 ||:
+  
+  for item in "crd/elasticsearches.logging.openshift.io" "crd/clusterloggings.logging.openshift.io" "ns/openshift-logging" "ns/openshift-operators-redhat"; do
+    oc delete $item --wait=false --ignore-not-found --force --grace-period=0
+  done
+  os::cleanup::all "${return_code}"
+  
+  exit ${return_code}
+}
+trap cleanup exit
+
+if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
+  source "$repo_dir/hack/common"
+fi
+if [ -n "${IMAGE_FORMAT:-}" ] ; then
+  IMAGE_CLUSTER_LOGGING_OPERATOR=$(sed -e "s,\${component},cluster-logging-operator," <(echo $IMAGE_FORMAT))
+else
+  IMAGE_CLUSTER_LOGGING_OPERATOR=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest}
```

Comment:
I think we are not using quay.io?  If not, use `registry.svc.ci.openshift.org/origin/4.2:cluster-logging-operator`

Reply:
I'll change it but seems we could simply drop this line as it should default to doing and using whats in the manifest